### ISSUE TITLE
The result in conversational task is better than paper's

### ISSUE THREAD

**Issue by C-hase** [wxl1999/UniCRS#4]
I ran your code successfully. Although the result in recommendation part is almost the same, the result of conversational part is so much better than your paper's. And I didn't change the source code. Would you tell me the reasons about it? 
The following is the result in my device.
 'test/dist@2': 0.5503033723719486, 'test/dist@3': 0.9362212501763792, 'test/dist@4': 1.211090729504727
The following is the result in your paper.
![image](https://user-images.githubusercontent.com/48970728/224596257-8e1c4a82-9814-4e0d-96aa-11d63595e69f.png)

**Comment by wxl1999** [comment]
@dayuyang1999 You are right. In addition, Dist is sensitive to the training steps. So I suggest paying more attention to human evaluation.
